test/clt-tests/buddy/test-fuzzy-search-negative.rec

```diff
@@ -17,16 +17,44 @@ ERROR 1064 (42000) at line 1: P01: syntax error, unexpected '-' near '-1'
 ––– input –––
 mysql -h0 -P9306 -e "SELECT * FROM name WHERE MATCH('SMITH') OPTION fuzzy='0';"
 ––– output –––
-ERROR 1064 (42000) at line 1: unknown option 'fuzzy'
+ERROR 1064 (42000) at line 1: Fuzzy plugin requires min_infix_len to be set
```

Comment:
Fuzzy search requires min_infix_len to be set in the table settings.

Reply:
Fixed in a separate PR